russian/call-ghostbusters/ghostbusters.md

```diff
@@ -0,0 +1,8 @@
+﻿Под звуки знаменитой песни, наигрывающей в голове, вы набираете 8-800-GHOSTBUSTERS и пытаетесь объяснить ситуацию.
+
+Похоже на проделки Зефирного человека! Охотники уже в пути! - слышите вы и облегчённо кладёте трубку.
+
+Что же делать теперь?
+
+[Откроете окно](../window/window.md)
+[Вернётесь ко сну.](../sleep/zefir.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.